app/(feeds)/layout.tsx

```diff
@@ -43,7 +43,7 @@ export default function FeedLayout({ children }: { children: ReactNode }) {
                   id="feed-container"
                 >
                   <div className="container mx-auto max-w-3xl space-y-6">
-                    {user && <CreatePost />}
+                    {!username && <CreatePost />}
```

Comment:
why are you removing check for `user`?
- actual: replaces existing check
- expected: extends existing check

Reply:
done